modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts

```diff
@@ -1,10 +1,33 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect<% if (feature) { %>, ofType<% } %> } from '@ngrx/effects';
-<% if (feature) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && !api) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && api) { %>import { catchError, map, switchMap } from 'rxjs/operators';<% } %>
+<% if (feature && api) { %>import { Load<%= classify(name) %>sFailure, Load<%= classify(name) %>sSuccess, <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
 
 @Injectable()
 export class <%= classify(name) %>Effects {
-<% if (feature) { %>
+<% if (feature && api) { %>
+
+  private _testObservable = of([
+    { myDataProperty: 1000, myOtherDataProperty: 'some-text' },
+    { myDataProperty: 1100, myOtherDataProperty: 'more-text' }]
+  )
+    .pipe(
+      delay(1000), // simulate latency of 1 second
+      // (_ => { throw new Error('oops'); }) // uncomment to test error handling
+    );
+
+  @Effect()
+  load<%= classify(name) %>s$ = this.actions$.pipe(
+    ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>),
```

Comment:
```suggestion
    ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name)s %>),
```

Reply:
this should be resolved now with the latest commit.